Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-core/kodi-main/) for Kodi core/kodi main.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-core/-/kodi-main/horizontal-auto.svg)

gade01 commented:
Still no good.

Including file locations and fuzzy translations.
Fix done at Weblate, but not deployed yet.

gade01 commented:
First PR where I don’t see any issues at all.

At the moment, it’s not possible to remove comments or untranslated strings.

@ronie care to take a look?

weblate committed:
Update translation files

Updated by "Source update" hook in Weblate.

Co-authored-by: Christian Gade <gade@kodi.tv>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/
Translation: Kodi core/kodi main
